integration-test/helpers/mocks.webkit.js

```diff
@@ -187,7 +208,38 @@ export function createWebkitMocks (platform = 'macos') {
         /** @type {null | Record<string, any>} */
         getAvailableInputTypes: null,
         storeFormData: null,
-        selectedDetail: null
+        selectedDetail: null,
+        /** @type {AskToUnlockProviderTypes | null} */
+        askToUnlockProvider: {
```

Comment:
Can you add a comment to explain why these 2, `askToUnlockProvider` & `checkCredentialsProviderStatus` are initialised with these defaults? they don't match the others above that tend to be empty/null until you've initialised them via the builder api. Are they here to prevent having to add them in every test, even unrelated ones? 

I don't have any issues with it, but I think a short comment would be good here :)

Reply:
In fact they were not needed, it was just leftover. I've removed them now. Thanks for catching this.